src/raven.js

```diff
@@ -455,12 +455,20 @@ Raven.prototype = {
      * @return {Raven}
      */
   captureMessage: function(msg, options) {
+    // Convert the message into a string
+    var exceptionMessage;
+    if (typeof msg === 'object' && msg !== null) {
+      exceptionMessage = JSON.stringify(msg);
+    } else {
+      exceptionMessage = '' + msg;
+    }
```

Comment:
Moved string convert logic before the ignore logic, so user's can ignore on custom objects